### PR TITLE
Create "Where to start with cloud migration?"

### PR DESCRIPTION
**Pull request by lukemorton**

### CODE REVIEW

File: posts/where_to_start_with_cloud_migration.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Not only are migrations not as simple as lift and shift, the prioritisation of what to move and when is fundamental to making a migration successful. Knowing how to phase the migration into many small steps is crucial for reducing risk.
+
+There are a number of questions that can help inform your roadmapping process.
+
+## What are your mission critical applications?
+
+Changes to mission critical applications will have significant risks associated, 
+will likely impact other applications, and will potentially be the most disruptive to the organisation. It is for these reasons that mission critical applications should be tackled first.
+
+If your business sells products and services that are self serviced by customers then your e-commerce system will likely be mission critical. Less obvious self service systems used by customers such as a trading platforms or bespoke software, perhaps even running in your customers premises, are also likely to be critical.
```
Comment (sowasred2012) [inline posts/where_to_start_with_cloud_migration.md]: The phrase "self sevice" in this context feels quite vague, I'm not clear what it means.

File: posts/where_to_start_with_cloud_migration.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## What are the goals behind your migration?
+
+Modernisation and cost savings are great supporting arguments for moving to the cloud but they aren't the only goals that should be driving your migration.
+
+In order to decide how to break the migration into reasonable chunks, and in order to be able to prioritise what to move first, goals that focus on improving the business value provided by existing applications is essential. I'll illustrate what I mean with a couple of examples.
+
+### Example one: Large monolithic business application
+
+Our first example is that of an application that is a businesses central database, provides workflows for business processes, is the CRM, and provides accounting tools for an organisation that is currently running on-premise. Clearly the organisation is tied into the application and as such a migration to the cloud could prove quite disruptive.
```
Comment (sowasred2012) [inline posts/where_to_start_with_cloud_migration.md]: businesses/business's

File: posts/where_to_start_with_cloud_migration.md (hunk 1 above)
Comment (RoryMacDonald) [inline posts/where_to_start_with_cloud_migration.md]: I'd reference something other than e-commerce here, as ecom is modern + 'web' era

File: posts/where_to_start_with_cloud_migration.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+# Where to start with cloud migration?
+
+Your organisation has decided it's ready to move to the cloud. Where do you start?
```
Comment (RyanMacG) [inline posts/where_to_start_with_cloud_migration.md]: Might just be me but this sentence seems a bit redundant with the title

**Pull request merged**